libs/postgres_ffi/src/lib.rs

```diff
@@ -184,6 +190,54 @@ pub fn fsm_logical_to_physical(addr: BlockNumber) -> BlockNumber {
     pages - 1
 }
 
+/// For some components, it's important to have better reproducible `initdb` command results.
+/// In particular, what pageserver parses from `initdb` output as `initdb_lsn`, is imporant to
+/// keep the same for the same branch.
+///
+/// Various env parameters influence that, same as various postgres versions may still change the
+/// values in the long run, but if this function was used, it's simpler to track such changes.
+///
+/// Ensures that there's a correct directory structure for initdb to run in.
+pub fn prepare_initdb_command(
+    pg_bin_dir: &Path,
+    pg_lib_dir: &Path,
+    target_dir: &Path,
```

Comment:
nit about API design: this takes three params, all with the same type. That's easy to mix up at the call site. Consider using builder pattern or introduce an `InitdbArgs` struct

Reply:
I have considered this, but have thought that it would overcomplicate the PR and left it out.
I still have some hope to merge it, so let's keep it as is for now, unless somebody else objects.

Reply:
Ok